.github/workflows/ci.yml

```diff
@@ -438,6 +460,54 @@ jobs:
           include-detectors: ${{ inputs.trufflehog-include-detectors }}
           exclude-detectors: ${{ inputs.trufflehog-exclude-detectors }}
 
+      - name: plugin-validator
+        if: ${{ inputs.run-plugin-validator == true }}
+        run: |
+          if [ -n "${PLUGIN_VALIDATOR_CONFIG_FILE}" ]; then
+            echo "Using provided plugin-validator configuration content."
+            echo "${PLUGIN_VALIDATOR_CONFIG_FILE}" > .plugin-validator.yaml
+            PLUGIN_VALIDATOR_CONFIG_PATH=.plugin-validator.yaml
+            cat ${PLUGIN_VALIDATOR_CONFIG_PATH}
+          elif [ -f "${PLUGIN_VALIDATOR_CONFIG_PATH}" ]; then
+            echo "Using plugin-validator configuration file at path: ${PLUGIN_VALIDATOR_CONFIG_PATH}"
+          fi
+
+          if [ ! -f "${PLUGIN_VALIDATOR_CONFIG_PATH}" ]; then
+            PLUGIN_VALIDATOR_CONFIG_PATH=".plugin-validator.yaml"
+            echo "⚠️ ${PLUGIN_VALIDATOR_CONFIG_PATH} configuration file is missing. Providing a default one as fallback."
```

Comment:
This should not really be a warning, I expect it'll be far more common to not pass your own config file.

Reply:
👍 , done!